newsroom/wire/search.py

```diff
@@ -128,7 +128,9 @@ def get_bookmarks_count(self, user_id):
         except Forbidden:
             return 0
 
-    def get_items(self, item_ids, size=None, aggregations=None, apply_permissions=False):
+    def get_items(
+        self, item_ids, size=None, aggregations=None, apply_permissions=False, sort_by=None, sort_order="desc"
```

Comment:
I would only use single `sort` param that would be used as is, this would be limiting to single sort field